src/test/components/Calculator.test.js

```diff
@@ -0,0 +1,9 @@
+import { render } from '@testing-library/react';
+import Calculator from '../../components/calculator';
+
+describe('Calculator', () => {
+  it('Renders calculator correctly', () => {
+    const { container } = render(<Calculator />);
+    expect(container).toMatchSnapshot();
+  });
+});
```

Comment:
- [ ] Great effort working on your testing, But when i run the test in my terminal all the test case passed except this particukar one and i would kindly suggest you work on it and be sure it passed the same way other tests passes.
Thank you.

Image of the error 👇 

![failed](https://github.com/Alejuda/Math-Magicians/assets/78634841/e25b3c21-4196-4006-bb7b-0f061ac271e5)

Reply:
Resolved now ✔️